### PR TITLE
upgrade: add runv upgrade guide

### PR DESCRIPTION
**Pull request by bergwolf**
runV and Kata Containers can live together but we need to make it clear
in the doc.

Fixes: #97

### CODE REVIEW

**Comment by jodh-intel** [comment]
Thanks @bergwolf.

lgtm

@egernst, @klynnrif - ptal.

File: Upgrading.md [hunk 1]
```diff
@@ -118,6 +119,15 @@ Follow one of the [installation guides](https://github.com/kata-containers/docum
 $ sudo docker run -ti busybox sh
 ```
 
+# Upgrade from runV
+
+runV and Kata Containers can run together on the same system without affecting each other, as long as they are
+not configured to use the same container root storage, which runV defaults to `/run/runv` and Kata Containers
```
Comment (klynnrif) [inline Upgrading.md]: Lines 125-126 suggested rewrite (to break up the sentence): not configured to use the same container root storage. Currently, runV defaults to `/run/runv` and Kata Containers defaults to `/var/run/kata-containers`.

File: Upgrading.md [hunk 2]
```diff
@@ -118,6 +119,15 @@ Follow one of the [installation guides](https://github.com/kata-containers/docum
...
+defaults to `/var/run/kata-containers`.
+
+Then to upgrade from runV, you just need to fresh install Kata Containers following one of
```
Comment (klynnrif) [inline Upgrading.md]: Now, to upgrade from runV you need to fresh install Kata Containers by following one of

**Comment by jodh-intel** [comment]
Hi @bergwolf - please could you update based on the review feedback so we can get this landed?

**Comment by jodh-intel** [comment]
Thanks @bergwolf.

lgtm

**Comment by jodh-intel** [comment]
\o/ - a green buttons and I'm pushing *two* of 'em ;)

**Comment by jodh-intel** [comment]
I lied - it was three.

**Pull request merged**